src/redux/sagas/bridge.js

```diff
@@ -52,21 +52,27 @@ function* withdrawWorker(action) {
 
 function* depositWorker(action) {
   try {
-    const { blockHash, status, events, errorData } = yield cps(bridgeDeposit, action.payload.values, action.payload.userWalletAddress);
+    const { blockHash, events, errorData } = yield cps(
+      bridgeDeposit,
+      action.payload.values,
+      action.payload.userWalletAddress,
+    );
     if (errorData.isError) {
       yield put(blockchainActions.setErrorExistsAndUnacknowledgedByUser.success(true));
       yield put(blockchainActions.setError.success(errorData));
       yield put(bridgeActions.deposit.failure(errorData));
-    }
-    else {
+    } else {
       const blockEvents = yield call(getBlockEvents, blockHash);
       const extrinsicIndex = events[0].phase.asApplyExtrinsic;
-      const receiptEventIndex = blockEvents.findIndex(({ phase, event }) =>
-        phase.isApplyExtrinsic && phase.asApplyExtrinsic.eq(extrinsicIndex) &&
-        event.method == "OutgoingReceipt"
+      const receiptEventIndex = blockEvents.findIndex(
+        ({ phase, event }) => phase.isApplyExtrinsic
+        && phase.asApplyExtrinsic.eq(extrinsicIndex)
+        && event.method === 'OutgoingReceipt',
       );
       const { amount, ethRecipient } = blockEvents[receiptEventIndex].event.data;
-      let receipt_id = subToEthReceiptId(blockHash, receiptEventIndex, amount, ethRecipient);
+      const receipt_id = subToEthReceiptId(blockHash, receiptEventIndex, amount, ethRecipient);
+      yield put(walletActions.getWallet.call());
```

Comment:
`put` this after the `bridgeActions.deposit` - otherwise the loader won't work as it should. (as the `deposit.success` will hide it before `getWallet` finishes).